client/blindbox/command/azure-sev/Dockerfile

```diff
@@ -7,8 +7,11 @@ ARG EMBED_IMAGE_TAR
 ENV DEBIAN_FRONTEND=noninteractive
 WORKDIR /root
 
+COPY ./attestation /root/attestation
+
 COPY ./sev-init.sh /root
 
+RUN chmod +x ./sev-init.sh
 RUN ./sev-init.sh
```

Comment:
Ok for now, but we should build what we need to only copy what's needed in the final image. This would also avoid to avoid build time dependencies in the final prod image.